src/dotnet/CarbonAware.WebApi/README.md

```diff
@@ -0,0 +1,74 @@
+# Carbon Aware REST API
+
+## Endpoints
+
+### POST /sci-scores
+
+This endpoint calculates the SCI score using the Green Software Foundation SCI specification formula.
+
+> ((E \* I) + M)/R
```

Comment:
I would recommend a link to some evergreen documentation. I think this one describes the exact formula, which will be helpful for future contributors.

https://greensoftware.foundation/articles/the-green-software-foundation-releases-alpha-version-of-software-carbon-intensity